oggm/tasks.py

```diff
@@ -21,7 +21,7 @@
 from oggm.core.climate import process_cru_data
 from oggm.core.climate import process_histalp_data
 from oggm.core.climate import process_custom_climate_data
-from oggm.core.climate import process_cesm_data
+from oggm.core.climate import process_gcm_data
```

Comment:
here you can add your new task as well: `from oggm.core.climate_prepro import process_cesm_data`